meta-mbl-distro/scripts/create-update-payload

```diff
@@ -74,7 +68,72 @@ import argparse_oe
 
 
 MBL_BOOT_ID = "BOOT"
-PAYLOAD_FORMAT_VERSION = b"1"
+MBL_APPS_ID = "APPS"
+MBL_ROOTFS_ID = "ROOTFS"
+
+PAYLOAD_FORMAT_VERSION = b"2"
+
+verbose_file = None
+
+def _strict_path_resolve(path):
+    """
+    Acts like "pathlib.Path(path).resolve(strict=True)" from Python >= 3.6.
+    """
+    # In python < 3.6, pathlib.Path.resolve() doesn't have a "strict"
+    # parameter, but it always does strict resolution so we just have to avoid
+    # passing the "strict" parameter.
+    resolve_args = {}
+    if "strict" in inspect.getfullargspec(path.resolve).args:
+        # In python >= 3.6, pathlib.Path.resolve() has a "strict"
+        # parameter, and we must set it to True to avoid the default
+        # value of False.
+        resolve_args = { "strict": True }
+    return pathlib.Path(path).resolve(*resolve_args)
```

Comment:
Does this work? I would've thought you'd need to use dictionary unpacking for this keyword argument i.e `(**resolve_args)` looks like you're just using list unpacking here so it'll just pass the key in.

Reply:
I tested it, and it did work... But then with list unpacking `"strict"` will be interpreted as the value for the `strict` parameter, and `"strict"` is truthy, so I suppose it would work by accident.

Will fix.

Reply:
But then why do I bother doing that at all when I could just do the following?
```Python
def _strict_path_resolve(path):
    """
    Acts like "pathlib.Path(path).resolve(strict=True)" from Python >= 3.6.
    """
    if "strict" in inspect.getfullargspec(path.resolve).args:
        # In python >= 3.6, pathlib.Path.resolve() has a "strict"
        # parameter, and we must set it to True to avoid the default
        # value of False.
        return pathlib.Path(path).resolve(strict=True)
    # In python < 3.6, pathlib.Path.resolve() doesn't have a "strict"
    # parameter, but it always does strict resolution.
    return pathlib.Path(path).resolve()
```

Reply:
Addressed in https://github.com/ARMmbed/meta-mbl/pull/730/commits/05190574fcf05e6b6d49b499666f85d53c147410